flink-kubernetes-operator-api/src/main/java/org/apache/flink/kubernetes/operator/api/spec/FlinkBlueGreenDeploymentConfigOptions.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.kubernetes.operator.api.spec;
+
+import org.apache.flink.configuration.ConfigOption;
+import org.apache.flink.configuration.ConfigOptions;
+
+import java.time.Duration;
+
+/** Configuration options to be used by the Flink Blue/Green Deployments. */
+public class FlinkBlueGreenDeploymentConfigOptions {
+
+    public static final String K8S_OP_CONF_PREFIX = "kubernetes.operator.";
+
+    public static final String BLUE_GREEN_CONF_PREFIX = K8S_OP_CONF_PREFIX + "bluegreen.";
+
+    public static ConfigOptions.OptionBuilder operatorConfig(String key) {
+        return ConfigOptions.key(BLUE_GREEN_CONF_PREFIX + key);
+    }
+
+    /**
+     * NOTE: The string durations need to be in format "{length value}{time unit label}", e.g.
+     * "123ms", "321 s". If no time unit label is specified, it will be considered as milliseconds.
+     * There is no fall back to parse ISO-8601 duration format, until Flink 2.x
+     *
+     * <p>Supported time unit labels are:
+     *
+     * <ul>
+     *   <li>DAYS： "d", "day"
+     *   <li>HOURS： "h", "hour"
+     *   <li>MINUTES： "m", "min", "minute"
+     *   <li>SECONDS： "s", "sec", "second"
+     *   <li>MILLISECONDS： "ms", "milli", "millisecond"
+     *   <li>MICROSECONDS： "µs", "micro", "microsecond"
```

Comment:
I am concerned about `µs` as it is a non standard character. [Flink docs](https://nightlies.apache.org/flink/flink-docs-release-1.20/docs/deployment/config/) use `us`. I suggest we use that here to be consistent.